test/real_content/test_swupd.bats

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env bats
+
+load "../functional/testlib"
+
+# Test swupd with real official content published on clearlinux.org
+# This test assumes all functional tests to be passing in order to work,
+# because we aren't going to test all swupd functionalities. The idea is
+# to test major swupd commands and standard use cases
+
+#TODO: Add test for crossing a format barrier
+
+# Optional parameters
+# $URL: Clearlinux content and version urls
+# $MAX_VERSIONS: The maximum number of versions we are going to be looking to
+#                test updates. Should be # at least 2 (current + update).
+#                Default is 10.
+# $BUNDLE_LIST: List of bundles to be installed on the system besides os-core
+#               and os-core-update. If empty, install all bundles.
+
+local SWUPD_OPTS # Swupd options to use in tests
```

Comment:
you are using local to define some variables here, but local is only valid within functions